src/Components/Molecules/FMViewPicture/FMViewPicture.interface.ts

```diff
@@ -0,0 +1,14 @@
+import { ReactChild } from "react";
+import { IFAAvatarProps } from "@Atoms/FAAvatar/FAAvatar.interface";
+import { IFAButtonProps } from "@Atoms/FAButton/FAButton.interface";
```

Comment:
don't use absolute path import. kindly update to relative import. https://hzngroup.slack.com/archives/C0318NE813M/p1645763123950249